src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -282,5 +286,45 @@ winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
         buffer = L"";
     }
 
-    return winrt::hstring{ buffer };
+    return buffer;
+}
+
+KeyChord KeyChordSerialization::FromString(const winrt::hstring& hstr)
+{
+    return _FromString(std::wstring{ hstr });
+}
+
+winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
+{
+    return hstring{ _ToString(chord) };
+}
+
+KeyChord ConversionTrait<KeyChord>::FromJson(const Json::Value& json)
+{
+    try
+    {
+        const std::string keyChordText{ json.isString() ?
+                                            json.asString() :
+                                            json[0].asString() };

```

Comment:
danger: what if someone does 
```json
"keys": { "foo": true, "bar": "you get the picture" }
```

I'd expect that doesn't throw `winrt::hresult_invalid_argument`